kubectl-metalk8s/cmd/kubectl-metalk8s.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"flag"
+	"github.com/spf13/pflag"
+
+	"github.com/scality/metalk8s/kubectl-metalk8s/pkg/cmd"
+
+	"k8s.io/klog"
+)
+
+func main() {
+	flags := pflag.NewFlagSet("kubectl-metalk8s", pflag.ExitOnError)
+	pflag.CommandLine = flags
+
+	klog.InitFlags(nil)
+
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
```

Comment:
There should be no need to do all of this when using Cobra, or at least, it can be done differently.

Reply:
To handle glog (klog) flags + Cobra one's I did not find any other method

Reply:
I'll create a fork to explain, will be easier than doing it here.

Reply:
https://github.com/scality/metalk8s/compare/improvement/GH-2498-simple-kubectl-plugin...user/nicolast/kubectl-plugin